pkg/apis/core/validation/validation.go

```diff
@@ -3413,6 +3415,8 @@ type PodValidationOptions struct {
 	AllowWindowsHostProcessField bool
 	// Allow more DNSSearchPaths and longer DNSSearchListChars
 	AllowExpandedDNSConfig bool
+	// Allow the invalid (but non-functional) seccomp annotations for backwards compatibility support.
+	AllowInvalidSeccompAnnotations bool
```

Comment:
I think we should drop this change and not relax validation in the future. The annotation validation will simply go unused as people stop specifying the annotations, but anyone who continues to set them will continue to have the same validation applied. (Dropping this makes this PR a lot smaller as well)

Reply:
Alright I dropped this part and reverted the tests.